libraries/AP_BattMonitor/AP_BattMonitor_GP9301.cpp

```diff
@@ -0,0 +1,87 @@
+#include "AP_BattMonitor_GP9301.h"
+#include <AP_HAL/AP_HAL.h>
+#include <GCS_MAVLink/GCS.h>
+
+/*
+  "battery" monitor for liquid fuel level systems that give a PWM value indicating quantity of remaining fuel.
```

Comment:
Inaccurate, but a "based on" so we know when we find bugs in the other driver we should look here too would be good :-)

In fact, most of the comments in here need adjustment.